Review README

cedricosei opened the pull request:
This update contains an initial review of the README files, which describe the main and subdirectories. In addition it contains a small adjustment in the license file (adding of year and names).

cedricosei committed:
Review README

This update contains an initial review of the README files, which describe the main and subdirectories. In addition it contains a small adjustment in the license file (adding of year and names).

jf87 commented on data/README.md:
Please check that the paths are correct. The first two ones are already not correct

jf87 commented:
Now the overall README and the data and src README.md files contain duplicated info. Please fix.

cedricosei committed:
Update README.md

cedricosei committed:
Update README.md

cedricosei committed:
Update README.md

jf87 commented on data/README.md:
```diff
@@ -1,2 +1,16 @@
+Below you will find a brief description of the specific subdirectories.
+
+
+- `./synthea_database` contains the Synthea data used as the basis for PostgreSQL, Graph DB (RDF), Neo4j, and MongoDB.
```
The paths are still wrong. Please check all things carefully before submitting it...

cedricosei committed:
Correction of the directories

cedricosei commented:
I updated the paths and changed the project README slightly such that we don't have the duplicate definition as mentioned above. I made sure to double check it more thoroughly this time.

jf87 commented on README.md:
```diff
@@ -24,27 +24,15 @@ This work will be presented at [NeurIPS 2024](https://neurips.cc/virtual/2024/po
 **Note: This repository will be updated further during the next week to make it easier to set up and use the four different databases.**
 
 ## Data
+The `./data` directory contains various used templates, synthea data, train and dev data and the compoted results.
```
Fix to:
The `./data` directory contains our template questions, the Synthea data, train and dev data, and the Text-to-Query results of our evaluated LLMs for all databases.

jf87 commented on README.md:
```diff
@@ -24,27 +24,15 @@ This work will be presented at [NeurIPS 2024](https://neurips.cc/virtual/2024/po
+For detailed information about the data presented in this project, please refer to the README in the `./data` directory. 
 
 ## Code
+The `./src` directory contains the code to reproduce the results presented in SM3-Text-to-Query research.
```
The `./src` directory contains the code to reproduce the results presented in SM3-Text-to-Query.

jf87 reviewed (changes requested):
see comments

cedricosei committed:
Updated to more accurate phrasing based on comments from Pull request

cedricosei commented:
Change requests were implemented as described in the comments above.

jf87 reviewed (approved):
Looks good. Thanks!
